### PR TITLE
Add KSQL DB support

### PR DESCRIPTION
**Pull request by burusha16**
## Issue : [#207](#207)

- added ksql db commands support
- `KSQL_DB` feature to features list
- added env variable description to readme
- unit tests

### Commands:
- create
- describe
- drop
- explain
- select
- show | list
- terminate

### CODE REVIEW

**Comment by iliax** [comment]
@burusha16 Why we dont use ksqldb java client (https://docs.ksqldb.io/en/latest/developer-guide/ksqldb-clients/java-client/) ?

File: kafka-ui-api/src/main/java/com/provectus/kafka/ui/model/Feature.java [hunk 1]
```diff
@@ -11,6 +11,7 @@
       .filter(Predicate.not(List::isEmpty))
       .isPresent()
   ),
+  KSQL_DB(cluster -> cluster.getKsqldbServer() != null),
```
Comment (germanosin) [inline kafka-ui-api/src/main/java/com/provectus/kafka/ui/model/Feature.java:14]: You should also add it here: 
https://github.com/provectus/kafka-ui/blob/master/kafka-ui-contract/src/main/resources/swagger/kafka-ui-api.yaml#L1371

**Review by germanosin — CHANGES_REQUESTED** [review]

File: kafka-ui-api/src/main/java/com/provectus/kafka/ui/model/Feature.java (hunk 1 above)
Comment (burusha16) [reply kafka-ui-api/src/main/java/com/provectus/kafka/ui/model/Feature.java:14]: already done

File: kafka-ui-api/src/main/java/com/provectus/kafka/ui/strategy/ksql/statement/BaseStrategy.java [hunk 2]
```diff
@@ -0,0 +1,165 @@
...
+  public KsqlCommand getKsqlCommand() {
+    return ksqlCommand;
+  }
+
+  public BaseStrategy ksqlCommand(KsqlCommand ksqlCommand) {
+    this.ksqlCommand = ksqlCommand;
+    return this;
+  }
+
+  protected KsqlCommandResponse serializeTableResponse(JsonNode response, String path) {
```
[the remark this answers is not in the document]
Comment (burusha16) [reply kafka-ui-api/src/main/java/com/provectus/kafka/ui/strategy/ksql/statement/BaseStrategy.java:50]: they both have same response item mapping, moved mapping to standalone method, now it's no logic duplication

**Review by germanosin — APPROVED** [review]

**Pull request merged**